doc/tutorials/introduction/biicode/tutorial_biicode.markdown

```diff
@@ -0,0 +1,157 @@
+@endcode
+
+Find your binaries in the bin folder:
+
+@code{.bash}
+$ cd bin
+$ ./diego_opencvex_main
+@endcode
+
+![](images/biiapp.png)
+
+@code{.bash}
+$ ./diego_opencvex_mainfaces
+@endcode
+
+![](images/bii_lena.png)
+
+###Developing your own application
+
+**biicode works with include headers in your source-code files**, it reads them and retrieves all the dependencies in its database. So it is as simple as typing:
+
+@code{.cpp}
+    #include "diego/opencv/opencv/cv.h"
+@endcode
+
+in the headers of your ``.cpp`` file.
+
+To start a new project using OpenCV, execute:
+
+@code{.bash}
+$ bii init mycvproject
+$ cd mycvproject
+@endcode
+
+The next line just creates a *myuser/myblock* folder inside "blocks" with a simple "Hello World" *main.cpp* into it. You can also do it manually:
+
+@code{.bash}
+$ bii new myuser/myblock --hello=cpp
+@endcode
+
+Now replace your *main.cpp* contents inside *blocks/myuser/myblock* with **your app code**.
+Put the includes as:
+
+@code{.cpp}
+    #include "diego/opencv/opencv/cv.h
+@endcode
+
+If you type:
+
+@code{.bash}
+$ bii deps
+@endcode
+
+You will check that ``opencv/cv.h`` is an "unresolved" dependency. You can find it with:
+
+@code{.bash}
+$ bii find
+@endcode
+
+Now, you can just `bii cpp:configure` and `bii cpp:build` your project as described above.
+
+**To use regular include directives**, configure them in your **biicode.conf** file. Let your includes be:
+
+@code{.cpp}
+    #include "opencv/cv.h"
+@endcode
+
+And write in your **biicode.conf**:
+
+@code{.cpp}
+    [includes]
+        opencv/cv.h: diego/opencv
+    [requirements]
+        diego/opencv: 0
+@endcode
+
+###Switching OpenCV versions
+
+If you want to try or develop your application against **OpenCV 2.4.10** and also against **3.0-beta**, change it in your **biicode.conf** file, simply alternating track in your `[requirements]`:
+
+@code{.cpp}
+    [requirements]
+        diego/opencv: 0
+@endcode
+
+replace with:
+
+@code{.cpp}
+    [requirements]
+        diego/opencv(beta): 0
+@endcode
+
+**Note** that the first time you switch to 3.0-beta, it will also take a while to download and build the 3.0-beta release. From that point you can change back and forth between versions, just modifying your *biicode.conf requirements*.
+
+The hooks used are in these blocks:
+* [OpenCV 2.4.10](http://www.biicode.com/diego/opencv)
+* [OpenCV 3.0 beta](http://www.biicode.com/diego/diego/opencv/beta)
+
+This is just an example of how can it be done with biicode python hooks. Probably now that CMake files reuse is possible with biicode, it could be better to implement it with CMake, in order to get more control over the build of OpenCV.
+
+Results and conclusion
+----------------------
+
+Installing OpenCV with biicode is straight forward for any OS.
+
+Run any example like you just did with *objdetect module* from OpenCV, or develop your own application. It only needs a *biicode.conf* file to get OpenCV library working in your computer.
```

Comment:
Here that link can be added: `*objdetect module*` -> `[objdetect module](@ref objdetect)`